ros2launch/ros2launch/api/api.py

```diff
@@ -159,6 +158,12 @@ def launch_a_launch_file(*, launch_file_path, launch_file_arguments, debug=False
     return ret
 
 
+def setup_security(*, keystore_dir: str, package_name: str):
+    os.environ['ROS_SECURITY_KEYSTORE'] = str(Path(keystore_dir).resolve()) if keystore_dir else ''
+    os.environ['ROS_SECURITY_ENABLE'] = 'true'
+    os.environ['ROS_SECURITY_STRATEGY'] = 'Enforce'
```

Comment:
Does this override existing environment variables? Do we an easy way to set `ROS_SECURITY_STRATEGY`?

Reply:
As far as I can tell the only way to enable security is via these environment variables

Reply:
I meant, are user-defined env var overridden? `export ROS_SECURITY_STRATEGY=NotEnforce ros2 launch foo bar --secure`. And if so, is there a way for the user to set `ROS_SECURITY_STRATEGY` ?

Reply:
Ah yeah then the user defined variables will be overridden. We could check if they're not set, and set them in that case.

That however opens up the door to having .bashrc or similar cause a node to unexpectedly not start

Reply:
yeah I'm not sure what's the ideal behavior if there are user-defined env vars, so I'd leave it like it is for now. However I think there should be a flag or such to set `ROS_SECURITY_STRATEGY` (e.g. `--security-strategy enforced`).

Reply:
Let's chat about this.

Reply:
We decided offline to leave these environment variables as an implementation detail, i.e. not part of the interface we expose here.